frontend/components/commons/results/RecordExtraActions.vue

```diff
@@ -102,7 +113,7 @@ export default {
       this.close();
     },
     close() {
-      this.open = false;
+      this.idState.open = false;
```

Comment:
We can also define computed props with setters, like doc [shows](https://vuejs.org/v2/guide/computed.html#Computed-Setter)

```javascript
// ...
computed: {
  open: {
    // getter
    get: function () {
      return this.idState.open
    },
    // setter
    set: function (newValue) {
       this.idState.open = newValue
    }
  }
}
// ...
```